aiokafka/protocol/metadata.py

```diff
@@ -187,34 +189,36 @@ class MetadataRequest_v0(Request):
     API_VERSION = 0
     RESPONSE_TYPE = MetadataResponse_v0
     SCHEMA = Schema(("topics", Array(String("utf-8"))))
-    ALL_TOPICS = None  # Empty Array (len 0) for topics returns all topics
+    ALL_TOPICS: Optional[int] = (
+        None  # Empty Array (len 0) for topics returns all topics
+    )
 
 
 class MetadataRequest_v1(Request):
     API_KEY = 3
     API_VERSION = 1
     RESPONSE_TYPE = MetadataResponse_v1
     SCHEMA = MetadataRequest_v0.SCHEMA
-    ALL_TOPICS = -1  # Null Array (len -1) for topics returns all topics
-    NO_TOPICS = None  # Empty array (len 0) for topics returns no topics
+    ALL_TOPICS: Optional[int] = -1  # Null Array (len -1) for topics returns all topics
+    NO_TOPICS: None = None  # Empty array (len 0) for topics returns no topics
```

Comment:
I don't see any use of these 2 fields. Should we consider removing them? On the other hand comments to them contain important information.

Reply:
Moved to docstring